bin/util.js

```diff
@@ -1,4 +1,48 @@
-const _ = require('lodash');
+const _ = require('lodash'),
+    ALL_CURL_OPTIONS = {
```

Comment:
@raghavbhatia336 Let's move these constant configs to a separate file, for easier maintainability.